com.bluejeans.BlueJeans.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Blue Jeans
+Comment=Blue Jeans Video Conferencing tool
+Exec=bluejeans %U
+Terminal=false
+X-MultipleArgs=false
```

Comment:
What uses this?

Reply:
Removed.

Reply:
I'm just wondering what it means, because if it means Exec doesn't take multiple args that is what `%u` means. Could you possibly test passing it multiple URLs?

Reply:
Hmm, to be honest, I am not sure what parameters it could take. :-) It seems that it doesn't support any switches. I have also tried passing the meeting URL (and multiple URLs) to it and nothing happened. (Both on the Flatpak and original non-Flatpak version.)

Reply:
(I have tried both `http(s)://` and `bjn(b)://` URLs.)

Reply:
Ok. :-) What about the `X-MultipleArgs`? Should I re-add it?